yt_dlp/extractor/nebula.py

```diff
@@ -95,35 +64,27 @@ def _fetch_nebula_bearer_token(self):
                                          note='Authorizing to Nebula')
         return response['token']
 
-    def _fetch_zype_access_token(self):
-        """
-        Get a Zype access token, which is required to access video streams -- in our case: to
-        generate video URLs.
-        """
-        user_object = self._call_nebula_api('https://api.watchnebula.com/api/v1/auth/user/', note='Retrieving Zype access token')
-
-        access_token = try_get(user_object, lambda x: x['zype_auth_info']['access_token'], str)
-        if not access_token:
-            if try_get(user_object, lambda x: x['is_subscribed'], bool):
-                # TODO: Reimplement the same Zype token polling the Nebula frontend implements
-                # see https://github.com/ytdl-org/youtube-dl/pull/24805#issuecomment-749231532
-                raise ExtractorError(
-                    'Unable to extract Zype access token from Nebula API authentication endpoint. '
-                    'Open an arbitrary video in a browser with this account to generate a token',
-                    expected=True)
-            raise ExtractorError('Unable to extract Zype access token from Nebula API authentication endpoint')
-        return access_token
-
-    def _build_video_info(self, episode):
-        zype_id = episode['zype_id']
-        zype_video_url = f'https://player.zype.com/embed/{zype_id}.html?access_token={self._zype_access_token}'
+    def _fetch_video_formats(self, slug):
+        stream_info = self._call_nebula_api(f'https://content.watchnebula.com/video/{slug}/stream/',
+                                            video_id=slug,
+                                            auth_type='bearer',
+                                            note='Fetching video stream info')
+        manifest_url = stream_info['manifest']
+        return self._extract_m3u8_formats_and_subtitles(manifest_url, slug)
+
+    def _build_video_info(self, episode, fetch_formats_and_subs=True):
+        if fetch_formats_and_subs:
+            fmts, subs = self._fetch_video_formats(episode['slug'])
+        else:
+            fmts, subs = [], []
         channel_slug = episode['channel_slug']
+        channel_title = episode['channel_title']
         return {
             'id': episode['zype_id'],
             'display_id': episode['slug'],
-            '_type': 'url_transparent',
-            'ie_key': 'Zype',
-            'url': zype_video_url,
+            'formats': fmts,
+            'subtitles': subs,
+            'url_result': f'https://nebula.app/{episode["slug"]}' if not fetch_formats_and_subs else '',
```

Comment:
```suggestion
            'url_result': f'https://nebula.app/{episode["slug"]}' if not fetch_formats_and_subs else '',
```
did u mean `webpage_url`?

Reply:
I guess I do.

I've always been using `url_result` here, I just looked up `webpage_url` for the first time. I guess `url_result` is a remnant of when this extractor was still handing off videos to the Zype extractor for downloading.

Changing it now.